bundles/org.eclipse.passage.lic.licenses.edit/src-gen/org/eclipse/passage/lic/licenses/edit/providers/GrantAcqisitionItemProvider.java

```diff
@@ -155,17 +155,6 @@ protected void addCreatedPropertyDescriptor(Object object) {
 						ItemPropertyDescriptor.GENERIC_VALUE_IMAGE, null, null));
 	}
 
-	/**
-	 * This returns GrantAcqisition.gif.
-	 * <!-- begin-user-doc -->
-	 * <!-- end-user-doc -->
-	 * @generated
```

Comment:
Should we preserve this method using "generated not"?

Reply:
There no images for these objects. Grant acquisition is runtime entity which is only to be persisted between sessions in order to restore the FLS runtime state (not implemented yet).